Meta/build-image-qemu.sh

```diff
@@ -182,9 +182,18 @@ cleanup() {
             if [ $USE_FUSE2FS -eq 1 ]; then
                 fusermount -u mnt || (sleep 1 && sync && fusermount -u mnt)
             else
-                umount mnt || ( sleep 1 && sync && umount mnt )
+                if [ "$(uname -s)" = "FreeBSD" ]; then
+                    ;
```

Comment:
Please fix the commit message to describe what problem is being worked around here.

Reply:
updated